### PR TITLE
Player Profile - VERSION 2.0

### PR DESCRIPTION
**Pull request by Maphesdus**

### CODE REVIEW

File: website/boxheadsweb/components/playerprofile/playerProfile.module.css [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+}
+
+.player button {
+  border: 2px solid white;
+  color: white;
+  border-radius: 0px;
+  background: none;
+  height: 40px;
+  width: 75px;
+}
```
Comment (ktwitchn) [inline website/boxheadsweb/components/playerprofile/playerProfile.module.css:51]: Same as the class above, also use variables for colors

File: website/boxheadsweb/components/playerprofile/playerProfile.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+      <form action="/url" method="GET">
+        <BoxheadsButton
+          navigateLink=""
+          title="vote"
+          onClick={() => {}}
+          className="actionClick"
+        ></BoxheadsButton>
+        <input type="text" placeholder="Notify by email" />
+      </form>
+    </div>
```
Comment (ktwitchn) [inline website/boxheadsweb/components/playerprofile/playerProfile.tsx:38]: Should we wrap this in a flex container?

File: website/boxheadsweb/components/playerprofile/playerProfile.tsx [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+import styles from "../playerProfile/playerProfile.module.css";
+import labelstyles from "../common/boxheadslabel.module.css";
+import BoxheadsButton from "../common/boxheadsbutton";
+import BoxheadsLabel from "../common/boxheadsLabel";
+
+function PlayerProfile() {
+  return (
+    <div className={styles.player}>
+      <Image
+        src="/images/player_profile_images/LebronPost.png"
```
Comment (ktwitchn) [inline website/boxheadsweb/components/playerprofile/playerProfile.tsx:12]: Add a prop on src so I can pass in different images based on which player I'm displaying

**Review by ktwitchn — COMMENTED** [review]
A good starting place for me to use in the vote page.

File: website/boxheadsweb/components/playerprofile/playerProfile.tsx (hunk 3 above)
Comment (Maphesdus) [reply website/boxheadsweb/components/playerprofile/playerProfile.tsx:12]: Add a prop on src? What does that mean?

Comment (ktwitchn) [reply website/boxheadsweb/components/playerprofile/playerProfile.tsx:12]: Sorry I should have said that better. Basically create a property for this component that allows me to update the src when I use the tag:

<playerProfile src="example.png"><playerProfile>

boxheadsButton does a similar thing with title, navigation link, etc.

**Comment by danielkboyer** [comment]
It's hard for me to review this because the result doesn't look like the figma result. This needs more work before it can be reviewed properly. 
Screenshot_20221113_075045
Screenshot_20221113_074946

**Review by danielkboyer — CHANGES_REQUESTED** [review]
Needs more work. Check comment above.

**Comment by Maphesdus** [comment]
> 

Thank you. I'll fix it tomorrow.

**Pull request closed without merging**